auth/src/test/java/com/firebase/ui/auth/testhelpers/FakeAuthResult.java

```diff
@@ -17,15 +17,11 @@
 import com.google.firebase.auth.AuthResult;
 import com.google.firebase.auth.FirebaseUser;
 
-public class FakeAuthResult implements AuthResult {
-    private FirebaseUser mFirebaseUser;
-
-    public FakeAuthResult(FirebaseUser firebaseUser) {
-        mFirebaseUser = firebaseUser;
-    }
+public enum FakeAuthResult implements AuthResult {
+    INSTANCE;
```

Comment:
That's just me being lazy since this is in a test. 😄 I can remove it if you want, but basically it's the easiest way of ensuring the singleton property since I'm using an Enum and I have a single instance of it.